core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -120,10 +133,17 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.navigateToTopicWithIndex = async function(index) {
-    var elem = await topicsListItems.get(index);
+    await waitFor.visibilityOf(topicsTable,
+      'Topic table taking too long to appear.');
+    var editTopicButton = element(
+      by.css('.protractor-test-edit-topic-button'));
```

Comment:
Can this be moved to the top of the file with the other selectors?

Reply:
Done.